CHANGES.txt

```diff
@@ -26,3 +26,6 @@ Fixes
 
 - Fixed an issue that caused a ``ClassCastException`` to be thrown when querying
   the ``sys.health`` table on a cluster that has ``blob`` tables.
+
+- Added support for views with aliased literals in select list / groupBy /
```

Comment:
as this is a fix, please rephrase it like "Fixed an issue when using aliased literals in..."